src/Components/ServiceScene/ServiceScene.tsx

```diff
@@ -81,6 +83,7 @@ export class ServiceScene extends SceneObjectBase<ServiceSceneState> {
     super({
       body: state.body ?? buildGraphScene(),
       $data: getQueryRunner(buildLokiQuery(LOG_STREAM_SELECTOR_EXPR)),
+      loading: true,
```

Comment:
instead of passing a new loading prop, couldn't we just grab the $data state from the ServiceScene in the child scene, and then use the `LoadingState` instead of a bool?

Reply:
This boolean is what makes the missing `loading` message work.

Reply:
The `loading` prop is only set to false when the data state is `LoadingState.Done` or `LoadingState.Error`, why do we need a new prop when we could check for these data states in the child?

Reply:
It is possible, but I don't see what it would solve. What would be the advantage of the children holding implementation details of the parent instead of exposing a bolean through the state. Seems like more propagation of an antipattern.

Reply:
In addition, these `loading` booleans also control the display of loading indicators.

Reply:
Gotcha, I was just thinking we should get the loading state directly from the queryRunner that's executing the query so there's a single source of truth, and then if we wanted to differentiate error states in the children, like error vs done vs loading we could, but I'll defer to your expertise on this since it sounds like you've dug in a bit more then I have :)

Reply:
It's not a bad idea at all, but right now I just don't see any advantage on changing it. I specially dislike having to know that your parent has a $data attribute that is the query and it has states, etc.